calculator.rb

```diff
@@ -0,0 +1,149 @@
+# Calculator program - Corinne Pingul
+
+# WELCOME----------------------------------------------------------------------
+puts """Hello there! I am a calculator, below are the operations I know how to compute: \n
+  ADDITION --> type \"add\" or \"+\"
+  SUBTRATION --> type \"sub\" or \"-\"
+  MULTIPLICATION --> type \"mult\" or \"*\"
+  DIVISION --> type \"div\" or \"/\"
+  POWERS --> type \"exp\" or \"^\"
+  """
+# QUESTIONS----------------------------------------------------------------------
+
+# Asks user what type of math they'd like to do
+print "What type of calculation would you like to do? "
+operation = gets.chomp.strip
+
+# Prints error message if user types something that's not an accepted operation
+while (operation != "add") && (operation != "+") && (operation != "sub") && (operation != "-") && (operation != "mult") && (operation != "*") && (operation != "div") && (operation != "/") && (operation != "exp") && (operation != "^")
```

Comment:
Even though this is a really long line, your (quite admirable) use of parens really helps keep it readable. :)

I dig the use of the `while` loop for verifying the user input. Good call there too. Moving forward, the Array `#include?` method will help you express this same thought in a more succinct way.